src/main/kotlin/racingcar/model/Game.kt

```diff
@@ -0,0 +1,35 @@
+package racingcar.model
+
+import camp.nextstep.edu.missionutils.Randoms
+
+class Game(carNames: List<String>) {
+    private val cars = carNames.map { Car(it) }
+
+    fun playRound() {
+        cars.forEach { car ->
+            if (isMoveForward()) {
+                car.move()
+            }
+        }
+    }
+
+    private fun isMoveForward(): Boolean {
+        return Randoms.pickNumberInRange(MIN_MOVE_THRESHOLD, MAX_MOVE_THRESHOLD) >= MOVE_THRESHOLD
+    }
+
+    fun getCars(): List<Car> {
+        return cars
+    }
+
+    fun getWinners(): List<String> {
+        val maxDistance = cars.maxOf { it.distance }
+
+        return cars.filter { it.distance == maxDistance }.map { it.name }
```

Comment:
바로 리턴해서 코드가 간결하게 표현하는 점도 좋지만, 이렇게 변수를 선언 후 값을 할당해서 리턴하는 방법을 통하면 좀 더 명확해지고, 디버깅시 이점이 있다고 합니다~!😀 아마도 취향차이이지 않으까 싶기도 하네요~
```suggestion
        val winners = cars.filter { it.distance == maxDistance }.map { it.name }
        return winners
```

Reply:
간결한 코드들은 바로 리턴해주는 습관이 있었는데 값을 할당하면 이런 장점들이 있군요 ㅎㅎ 이런 부분도 더 신경을 써봐야겠네요
감사합니다 !